src/main/java/main/movie/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package main.movie.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import main.movie.dao.OrderDao;
+import main.movie.lib.Inject;
+import main.movie.lib.Service;
+import main.movie.model.Order;
+import main.movie.model.ShoppingCart;
+import main.movie.model.Ticket;
+import main.movie.model.User;
+import main.movie.service.OrderService;
+import main.movie.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(User user) {
```

Comment:
let's better pass shopping cart here(you can get tickets and user from it without sending requests to DB)